eva/parser/types.py

```diff
@@ -62,6 +62,7 @@ class FileFormatType(EVAEnum):
     VIDEO  # noqa: F821
     CSV  # noqa: F821
     IMAGE  # noqa: F821
+    PDF
```

Comment:
You will need to run the linter -- 
`bash script/test/test.sh -m "LINTER"`  
`bash script/test/test.sh -m "TEST"  `

Reply:
Sure, I will do it

Reply:
I had run the linter, which got executed successfully 
and from the below image we can see that the pdf is been added to the eva and it has been throwing error at the executor because as of now we haven’t provided any executor. The PDFexecutor is in progress and will be developed soon
<img width="526" alt="image" src="https://github.com/georgia-tech-db/eva/assets/42251896/8c9933f9-b7c3-4a8b-8046-995341cfd432">